sdk/logical/schedule.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package logical
```

Comment:
Common helpers to parse Rotation Schedules. Will be used by all plugins, hence added to CE to avoid SDK splitting

Reply:
Same comment on putting this all in `logical`. Also, will plugin authors actually need any of these? These seems like they will be internal to the rotation manager but maybe I am not seeing it all clearly?

Reply:
Good point. While the helpers in `rotation_job` are used by plugins, the helpers in `schedule.go` are currently only internally used by the `rotation_job` code, and not plugins themselves.

The idea behind having these be shareable for plugins was because this is essentially the same code (or an extension of it) as the one in [database/schedule/schedule.go](https://github.com/hashicorp/vault/blob/main/builtin/logical/database/schedule/schedule.go). Ideally down the line, we would unify both implementations to use the same package when we work on Automated Static Role Rotation, and so we preemptively included this in the `sdk` package and made it shareable. Not particularly stuck on either idea though, so whatever folks think is the better way forward :)

Reply:
Left it shareable in the `sdk/rotation` package for now